install_pim/docker/installation_docker.rst

```diff
@@ -29,15 +29,18 @@ Configuring you package manager
 
 The default values defined in `docker-compose.yml` match the default values for the `composer` and `yarn` directories:
  - Composer: `~/.composer`
```

Comment:
This one could be set in `~/.config/composer` to match the cache directory, but it is not mandatory.